pkgs/tools/misc/colorz/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "colorz";
+  version = "1.0.3";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0ghd90lgplf051fs5n5bb42zffd3fqpgzkbv6bhjw7r8jqwgcky0";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ pillow scipy ];
+
+  meta = with lib; {
+    description = "Color scheme generator";
+    homepage = https://github.com/metakirby5/colorz;
```

Comment:
```suggestion
    homepage = "https://github.com/metakirby5/colorz";
```

Reply:
oh, ok. I didn't use strings for the homepage url since this isn't done in examples from th e nixpkgs manual either https://nixos.org/nixpkgs/manual/#chap-meta

Reply:
oh, we need to change that, an rfc was merged recent